i18n/dnd5e/gl/dnd5e.json

```diff
@@ -0,0 +1,758 @@
+  "DND5E.AbilityChaAbbr": "car",
+  "DND5E.AbilityModifier": "Modificador de Atributo",
+  "DND5E.AbilityPromptText": "Que tipo de rolada de {ability}?",
+  "DND5E.AbilityPromptTitle": "Proba de Atributo de {ability}",
+  "DND5E.AbilityUseHint": "Configura como desexas usar o(a) {type} {name}.",
+  "DND5E.AbilityUseUnavailableHint": "Non hai máis usos para este item!",
+  "DND5E.AbilityUseChargedHint": "Este {type} está cargado e pronto para usar!",
+  "DND5E.AbilityUseRechargeHint": "Este(a) {type} está esgotado e debe ser recargado!",
+  "DND5E.AbilityUseNormalHint": "Este(a) {type} ten {value} de {max} usos e {per} restantes.",
+  "DND5E.AbilityUseConsumableChargeHint": "O uso deste(a) {type} consumirá unha carga de {value} restante(s).",
+  "DND5E.AbilityUseConsumableQuantityHint": "Usando este(a) {type} consumirás unha cantidade de {quantity} restante(s)",
+  "DND5E.AbilityUseConsumableDestroyHint": "O uso deste(a) {type} consumirá a sua carga final e será destruído.",
+  "DND5E.AbilityUseConsume": "Consumir uso disponíbel?",
+  "DND5E.AbilityUseChargesLabel": "{value} Cargas",
+  "DND5E.AbilityUseConsumableLabel": "{max} por {per}",
+  "DND5E.AbilityUseCast": "Conxurar Maxia",
+  "DND5E.AbilityUseUse": "Usar Habilidade",
+  "DND5E.Action": "Acción",
+  "DND5E.ActionPl": "Accións",
+  "DND5E.ActionAbil": "Proba de Atributo",
+  "DND5E.ActionHeal": "Cura",
+  "DND5E.ActionMSAK": "Ataque máxico corpo a corpo",
+  "DND5E.ActionMWAK": "Ataque armado corpo a corpo",
+  "DND5E.ActionOther": "Outro",
+  "DND5E.ActionRSAK": "Ataque máxico a distancia",
+  "DND5E.ActionRWAK": "Ataque armado a distancia",
+  "DND5E.ActionSave": "Salvagarda",
+  "DND5E.ActionUtil": "Utilidade",
+  "DND5E.ActionWarningNoItem": "O item solicitado {item} xa non existe no Actor {name}",
+  "DND5E.ActionWarningNoToken": "Debes controlar ou mais Tokens para usar esta opción.",
+  "DND5E.Add": "Crear",
+  "DND5E.Advantage": "Vantaxe",
+  "DND5E.Alignment": "Aliñamento",
+  "DND5E.AlignmentCE": "Caótico malo",
+  "DND5E.AlignmentCG": "Caótico bo",
+  "DND5E.AlignmentCN": "Caótico neutro",
+  "DND5E.AlignmentLE": "Legal malo",
+  "DND5E.AlignmentLG": "Legal bo",
+  "DND5E.AlignmentLN": "Legal neutro",
+  "DND5E.AlignmentNE": "Legal malo",
+  "DND5E.AlignmentNG": "Neutro bo",
+  "DND5E.AlignmentTN": "Neutro",
+  "DND5E.Appearance": "Apariencia",
+  "DND5E.Attunement": "Sintonización",
+  "DND5E.AttunementNone": "Sintonización non necesaria",
+  "DND5E.AttunementRequired": "Sintonización necesaria",
+  "DND5E.AttunementAttuned": "Sintonizado",
+  "DND5E.Attuned": "Sintonizado",
+  "DND5E.ArmorClass": "Clase de Armadura",
+  "DND5E.AC": "CA",
+  "DND5E.Attack": "Ataque",
+  "DND5E.AttackPl": "Ataques",
+  "DND5E.AttackRoll": "Rolada de Ataque",
+  "DND5E.Attributes": "Atributos",
+  "DND5E.Background": "Antecedente",
+  "DND5E.Biography": "Biografía",
+  "DND5E.Bonds": "Vínculos",
+  "DND5E.BonusAbilityCheck": "Bonus Global para Probas de Atributo",
```

Comment:
Coido que bonus está traducido como bonificador nos outros módulos (o mesmo nas outras liñas).

Reply:
Correcto, con un matiz. En D&D temos a "Bonus Action", traducida en pt_BR cono "Ação Bonus", que hai que ter coidado como se traduce, porque se usamos "acción adicional", "acción bonificada" ou cousas así máis naturais, podemos causar confusión con certas regras.